server/index.js

```diff
@@ -10,19 +11,19 @@ const wrap = fn => (...args) => fn(...args).catch(args[2]);
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
-app.use(express.static(__dirname + '/../client/dist'));
+app.use(express.static(path.join(__dirname, '..', 'client', 'dist')));
 
 // swagger
 app.use('/docs', swaggerUi.serve, swaggerUi.setup(require('../swagger.json')));
 
 // categories
-app.get('/categories', wrap(async (req, res) => {
+app.get('/api/categories', wrap(async (req, res) => {
   const categories = await db.Category.findAll();
   res.json(categories);
 }));
 
 // events
-app.get('/events', wrap(async (req, res) => {
+app.get('/api/events', wrap(async (req, res) => {
   const { categoryId } = req.query;
   // TODO: limit events to last X days
```

Comment:
Only comment here is that pretty soon we're going to want to start limiting to the last several days, and ordering the response to the client in the most helpful way for rendering.  Alternatively, we can get the caching layer set up and deal with it there?

Reply:
@michellelockett yeah! lets try to make dat cache